### ISSUE TITLE
The vandermonde matrix is not invertible?

### ISSUE THREAD

**Issue by lpstudy** [drmingdrmer/lrc-erasure-code#2]
![image](https://user-images.githubusercontent.com/2081195/69023984-ca147100-09fb-11ea-8aeb-401e1cdff08b.png)

Intel ISA-L told that using this way to construct Vandermonde matrix does not guarantee the invertible property for the sub matrix. I hope to get help from you. 

Do I make a mistake or in your condition, this works well engough?

**Comment by drmingdrmer** [comment]
Yes Vandermonde matrix does not guarantee invertable for sub matrix.
Use the Cauchy matrix instead. 

> Do I make a mistake or in your condition, this works well engough?

I do not get what you mean about the "mistake".
Here I have a short explain about the invertibility which might help you a bit: http://drmingdrmer.github.io/tech/distributed/2017/02/01/ec.html#vandermonde-%E7%9F%A9%E9%98%B5%E7%9A%84%E5%8F%AF%E9%80%86%E6%80%A7
